swift/src/IceImpl/ObjectAdapter.mm

```diff
@@ -223,6 +223,8 @@ - (dispatch_queue_t)getDispatchQueue:(NSError* _Nullable* _Nullable)error
 {
     try
     {
+        // Swift always sets InitializationData.useDispatchQueueExecutor to true
```

Comment:
It's odd the return value is not nullable like elsewhere. Why this difference?

Reply:
https://developer.apple.com/documentation/swift/about-imported-cocoa-error-parameters

I think nullable would be correct here.